src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,541 @@
+event handlers will fire.
+
+Next, the event will _propagate_ to the `#A` element. Again, the browser will ask: "Is anything
+listening for a click event on `#A` in the capturing phase and if so, the appropriate event handlers
+will fire.
+
+Next, the event will _propagate_ to the `#B` element (and the same question will be asked).
+
+Finally, the event will reach its target and the browser will ask: "Is anything listening for a
+click event on the `#C` element in the capturing phase?" The answer this time is "yes!" This brief
+period of time when the event is _at_ the target, is known as the "target phase." At this point, the
+event handler will fire, the browser will console.log "#C was clicked" and then we're done, right?
+_Wrong!_ We're not done at all. The process continues, but now it changes to the bubbling phase.
+
+#### Event Bubbling
+
+The browser will ask:
+
+"Is anything listening for a click event on `#C` in the bubbling phase?" Pay close attention here.
+It is completely possible to listen for clicks (or any event type) in _both_ the capturing _and_
+bubbling phases. And if you had wired up event handlers in both phases (e.g. by calling
+`.addEventListener()` twice, once with `capture = true` and once with `capture = false`), then yes,
+both event handlers would absolutely fire for the same element. But it's also important to note that
+they fire in different phases (one in the capturing phase and one in the bubbling phase).
+
+Next, the event will _propagate_ (more commonly stated as "bubble" because it seems as though the
+event is traveling "up" the DOM tree) to its parent element, `#B`, and the browser will ask: "Is
+anything listening for click events on `#B` in the bubbling phase?" In our example, nothing is, so
+no handlers will fire.
+
+Next, the event will bubble to `#A` and the browser will ask: "Is anything listening for click
+events on `#A` in the bubbling phase?"
+
+Next, the event will bubble to `<body>`: "Is anything listening for click events on the `<body>`
+element in the bubbling phase?"
+
+Next, the `<html>` element: "Is anything listening for click events on the `<html>` element in the
+bubbling phase?
+
+Next, the `document`: "Is anything listening for click events on the `document` in the bubbling
+phase?"
+
+Finally, the `window`: "Is anything listening for click events on the window in the bubbling phase?"
+
+Phew! That was a long journey, and our event is probably very tired by now, but believe it or not,
+that is the journey every event goes through! Most of the time, this is never noticed because
+developers typically are only interested in one event phase or the other (and typically it is the
+bubbling phase).
+
+It's worth spending some time playing around with event capturing and event bubbling and logging
+some notes to the console as handlers fire. It's very insightful to see the path that an event
+takes. Here is an example that listens to every element in both phases.
+
+```html
```

Comment:
Is this better as an embedded demo that users can click on?

Reply:
@sstchur has a demo up internally that I have suggested he move to Glitch.

Reply:
No problem, I will take a look at moving to Glitch

Reply:
I've added a link at the end to a Glitch project I setup which has all the demos.  Lmk what you think?